api/types.go

```diff
@@ -166,16 +174,22 @@ type ShowResponse struct {
 }
 
 type CopyRequest struct {
-	Source      string `json:"source"`
+	Source string `json:"source"`
+	Target string `json:"target"`
+
+	// Destination is deprecated, see Target
 	Destination string `json:"destination"`
```

Comment:
`destination` works here. E.g. Apple often calls it "Destination" and it will avoid a deprecation